packages/css/src/grid/grid-cell.scss

```diff
@@ -5,14 +5,20 @@
 
 @import "../../utils/breakpoint";
 
-@media screen and (width > $amsterdam-breakpoint) {
-  @for $i from 1 through 12 {
-    .amsterdam-grid-column-start-#{$i} {
-      grid-column-start: $i;
-    }
+.amsterdam-grid-cell:first-child {
+  grid-column-start: content;
```

Comment:
This is an attempt to skip the margin column if the first grid item doesn’t have an explicit grid column specified – having implicit grid placement working as expected would make it much easier for implementers.

It]s rather feeble though – e.g. the footer has a `VisuallyHidden` title as its first child, which renders this approach useless.

Reply:
We found this approach to be lacking.